modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl

```diff
@@ -1136,72 +1137,10 @@ public abstract class Base${schemaName}ResourceImpl
 			List<ResourceAction> resourceActions = resourceActionLocalService.getResourceActions(resourceName);
 
 			if (Validator.isNotNull(roleNames)) {
-				return Page.of(actions, _getPermissions(contextCompany.getCompanyId(), resourceActions, id, resourceName, StringUtil.split(roleNames)));
+				return Page.of(actions, PermissionUtil.getPermissions(contextCompany.getCompanyId(), resourceActions, id, resourceName, StringUtil.split(roleNames)));
```

Comment:
@carolmariaabb we can't do this. REST Builder generated code needs to be backwards compatible all the way back to 7.1. If someone on an earlier version runs REST Builder now it will generate this code but they will not have the Util method and it will not compile. @ccorreagg fyi.

Reply:
@4lejandrito we changed the approach to keep the `_getPermissions` method here.

Reply:
oh ok! thanks!